src/BizHawk.Client.EmuHawk/RetroAchievements/RCheevos.cs

```diff
@@ -414,7 +414,7 @@ public override void Restart()
 
 			// validate addresses now that we have cheevos init
 			// ReSharper disable once ConvertToLocalFunction
-			LibRCheevos.rc_runtime_validate_address_t peekcb = address => _readMap.ContainsKey(address);
+			bool peekcb(int address) => _readMap.ContainsKey(address);

```

Comment:
This should not be changed. Passing in `peekcb` as a local function will create a Delegate in _lib.rc_runtime_validate_addresses, which actually allows it to be GC'd. Doing it as a local variable prevents that.

Reply:
agree I'll revert and pragma this